### ISSUE TITLE
Package "php-smbclient" is not compatible with PHP 7.2

### ISSUE THREAD

**Issue by maximilian-walter** [oerdnj/deb.sury.org#815]
The package is not compatible with PHP 7.2:

```bash
$ apt-cache show php-smbclient
Package: php-smbclient
[...]
Architecture: amd64
Version: 0.9.0-1~ubuntu16.04.1+deb.sury.org+1
Provides: php-libsmbclient, php5.6-smbclient, php7.0-smbclient, php7.1-smbclient
Depends: ucf, php-common (>= 1:7.0+33~), phpapi-20160303 | phpapi-20151012 | phpapi-20131226, libc6 (>= 2.14), libsmbclient (>= 2:4.1.1)
[...]
```

It would be great if this extension could be used also with PHP 7.2.

Let me know if this is a problem of the extension itself and if this is the wrong bug tracker for this request. :-)

Thanks!

**Comment by joeyhub** [comment]
I have the same problem.

pecl happily builds it so there's no obvious reason for it to be missing.

**Comment by diabollo1** [comment]
On what architecture?
Maybe it's the same problem as here:
https://github.com/oerdnj/deb.sury.org/issues/579

**Comment by maximilian-walter** [comment]
No, i'm on a regular x86_64 system (Ubuntu 16.04 LTS)

**Comment by maximilian-walter** [comment]
I close this issue, because the package is now also available for PHP 7.2. Thanks!
